easybuild/easyconfigs/b/bam-readcount/bam-readcount-0.7.4-cmake_install_prefix.patch

```diff
@@ -0,0 +1,12 @@
+diff -u bam-readcount-0.7.4/CMakeLists.txt bam-readcount-0.7.4.new/CMakeLists.txt
```

Comment:
@verdurin please include a line or two of comment above, like:

```
patch out hardcoded installation prefix
author: Adam Huffman (<affiliation>)
```

(where you replace `<affiliation>`, of course :))